src/server/templates/page.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function page({ body = "", assets, title }) {
```

Comment:
I generally prefer to use named functions on the top level variables of the scope, to have some names to refer to in case of having to debug, etc.

If you have strong opinions I don't mind changing it.

---

Re docs, sorry I totally forgot, I was on bootstrapping mode. Addressed here e867bd0. Saw the linting fail as I was adding the docs 👌